SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRepNotify/SpatialTestRepNotify.cpp

```diff
@@ -151,12 +151,20 @@ void ASpatialTestRepNotify::PrepareTest()
 			}
 
 			// We consciously differ from native UE here
-			if (GetNetDriver()->IsA(USpatialNetDriver::StaticClass()))
+			// Also, the native behaviour changed when going from 4.25 to 4.26.
+			// On older versions, we expect the old array to have 3 elements, but on Spatial and on native starting from 4.26, we expect 2 elements.
+#if ENGINE_MINOR_VERSION >= 26
```

Comment:
We need Miron's future fix for this test to pass

Reply:
You mean for it to pass on 4.25?